src/module-1/cutStrings/index.js

```diff
@@ -1,10 +1,9 @@
 export const cutStrings = (arr = []) => {
-	let min = Infinity;
+	const lenItem = [];
```

Comment:
to get array with strings lengths, you could just write
`arr.map(item => item.length)`
looping through each record in array and pushing it to new array is heavier than `map`.